examples/python/notebook/cube.ipynb

```diff
@@ -298,7 +298,7 @@
     "\n",
     "bp = rrb.Blueprint(collapse_panels=True)\n",
     "\n",
-    "rec.notebook_show(blueprint=bp)"
+    "rec.notebook_show(blueprint=bp)\n"
```

Comment:
A lot of new `\n` in this file O.o

Reply:
That's how VSCode formatted it when I saved the file. Seems fine